gtsam/discrete/tests/testAlgebraicDecisionTree.cpp

```diff
@@ -48,7 +48,7 @@ template<> struct traits<ADT> : public Testable<ADT> {};
 #define DISABLE_DOT
 
 template<typename T>
-void dot(const T&f, const string& filename) {
+void DOT(const T&f, const string& filename) {
```

Comment:
No, no need to do this. Fully qualify other by doing `gtsam::dot`

Reply:
That doesn't work since the compiler thinks that this `dot` is the Vector dot. Though I may be able to resolve this once the equality method is figured out.

Reply:
Resolved